catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -1004,34 +1002,34 @@ public CreateResponse create(CreateRequest createRequest)
                 }
                 metacard.setAttribute(new AttributeImpl(Metacard.SECURITY, itemPolicyMap));
             }
-            createReq.getProperties()
+            createRequest.getProperties()
                     .put(PolicyPlugin.OPERATION_SECURITY, requestPolicyMap);
 
             for (AccessPlugin plugin : frameworkProperties.getAccessPlugins()) {
-                createReq = plugin.processPreCreate(createReq);
+                createRequest = plugin.processPreCreate(createRequest);
```

Comment:
The problem I have with that is that createReq = createRequest means that createReq is pointing to the same object createRequest is. That means all the stuff that's being done to createReq is also being done to createRequest. I think the original author thought he was making a copy, but really he made a copy of the reference.

It's even more confusing because then createReq is getting reassigned and manipulated throughout the function, so all of this stuff is happening to the object that both variables are pointing to until you get to L1019 and it gets re-assigned and is actually pointing to something else. 

Using a single variable makes it clear that that variable is the thing being manipulated. There is also so much going on that I don't know if we want side effects or not. If we don't, should I make a proper copy?